### ISSUE TITLE
volume.Request.MountId is Empty

### ISSUE THREAD

**Issue by davidstack** [docker/go-plugins-helpers#52]
when i write my volume driver for docker, i can not get the MountId when mount volume.my code is 

```
func (driver *LvmPersistDriver) Mount(req volume.Request) volume.Response {
	driver.Mutex.Lock()
	defer driver.Mutex.Unlock()
	fmt.Println("Mount Called... ")
	devicename := driver.Volumes[req.Name]
	if devicename == "" {
		return volume.Response{Err: fmt.Sprintf("The volume %s not exist", req.Name)}
	}
	fmt.Println(devicename)

	mountId := req.MountID
.....
```
the req.MountID is always empty, is there anything wrong with my code?thanks

**Comment by davidstack** [comment]
my docker version is 1.11
```
Client:
 Version:      1.11.2
 API version:  1.23
 Go version:   go1.5.4
 Git commit:   b9f10c9
 Built:        Wed Jun  1 21:47:50 2016
 OS/Arch:      linux/amd64

Server:
 Version:      1.11.2
 API version:  1.23
 Go version:   go1.5.4
 Git commit:   b9f10c9
 Built:        Wed Jun  1 21:47:50 2016
 OS/Arch:      linux/amd64
```

**Comment by cpuguy83** [comment]
Support for this is only available from 1.12.

**Comment by govint** [comment]
Tested on Docker 1.12 getting an empty MountID as well. Will verify again.

**Comment by dave-tucker** [comment]
Is this closed by #56?

**Comment by govint** [comment]
I need to rebase with the changes for PR 54 and then merge.

**Comment by govint** [comment]
Yes, should be closed by #56
